neuro-cli/src/neuro_cli/image.py

```diff
@@ -137,9 +149,21 @@ async def ls(
     with root.status("Fetching images"):
         images = await root.client.images.list(cluster_name=cluster)
 
-    if project:
-        projects = set(project)
-        images = [image for image in images if image.project_name in projects]
+    if all_orgs:
```

Comment:
Our image name parser works incorrect - it threats org names as project names and the underlying project name as a part of image name.
AFAIU, this affects only the representation (and filtering here), so we could fix it later / in a separate PR.

example:
(current config: testproj + no-org, testorg is an org with two projects, yevheniisemendiak and testproj and tp4 is a project w.o. org the current user has access to)
![image](https://github.com/neuro-inc/neuro-cli/assets/32543098/addf5d32-e072-45dc-95cc-f856c99f5c40)

Reply:
I will do it in a separate pr. AFAIU it was broken when we introduced orgs, parser doesn't recognize non current orgs and treats them as project.